shared/src/utils/governanceMath.ts

```diff
@@ -114,3 +115,116 @@ export const calculateBoostedRewards = (
     ? baseRewardsPercentage * boostedMultiplier - baseRewardsPercentage
     : 0;
 };
+
+interface ClaimableRbnCalculationProps {
+  periodTimestamp: number; // period timestamp of current period, LG5.period_timestamp[period]
+  integrateInvSupply: BigNumber; // integrate_inv_supply of the current period, LG5.integrate_inv_supply[period]
+  integrateFraction: BigNumber; // LG5.integrate_fraction[addr]
+  integrateInvSupplyOf: BigNumber; // LG5.integrate_inv_supply_of[addr]
+  futureEpochTime: number; // LG5.future_epoch_time
+  inflation_rate: BigNumber; // LG5.inflation_rate
+  minterRate: BigNumber; // MinterContract.rate
+  isKilled: boolean; // LG5.is_killed
+  workingSupply: BigNumber; // LG5.working_suppply
+  workingBalance: BigNumber; // LG5.working_balances[addr]
+  mintedRBN: BigNumber; // amount of RBN minted. From MinterContract.minted[account][gauge]
+  gaugeContractAddress: string;
+  gaugeControllerContract: LiquidityGaugeController;
+}
+
+export const calculateClaimableRbn = async ({
```

Comment:
Can we write a unit test for this function? We could mock the gauge weight calls